examples/non-kubernetes/aws-otel-config-file.yaml

```diff
@@ -1,21 +1,39 @@
 extensions:
   health_check:
 receivers:
+  awsxray:
+    endpoint: 0.0.0.0:2000
+    transport: udp
+  jaeger:
+    protocols:
+      thrift_compact:
```

Comment:
Do we need all those formats supported? Some of them are considered deprecated AFAIK

Reply:
It is still possible to use exporters for this formats. I would leave them. Can be removed in the future.